app/views/feedbacks/_form.html.erb

```diff
@@ -1,48 +1,48 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-l">Give feedback on this service</h1>
-    <%= form_for feedback do |f| %>
-      <%= GovukElementsErrorsHelper.error_summary f.object, 'There is a problem', '' %>
+    <%= govuk_form_for feedback do |f| %>
+      <%= f.govuk_error_summary %>
       <p>
         Help us improve the school experience service.
       </p>
 
       <%= f.hidden_field :referrer %>
 
-      <%= f.radio_button_fieldset :reason_for_using_service do |fieldset| %>
-        <% f.object.reasons_for_using_service.each do |option| %>
+      <%= f.govuk_radio_buttons_fieldset :reason_for_using_service do %>
+        <% @feedback.reasons_for_using_service.each do |option| %>
           <% if f.object.requires_explanation? option %>
-            <%= fieldset.radio_input option do %>
-              <%= fieldset.text_area :reason_for_using_service_explanation %>
+            <%= f.govuk_radio_button :reason_for_using_service, option, link_errors: true do %>
+              <%= f.govuk_text_area :reason_for_using_service_explanation %>
             <% end %>
           <% else %>
-            <%= fieldset.radio_input option %>
+            <%= f.govuk_radio_button :reason_for_using_service, option, link_errors: true %>
```

Comment:
A note here, you just want `link_errors: true` on the first radio/checkbox in a fieldset (as this will be the one the error summary will link to if there's an error on the field)

Reply:
Cheers, I changed it so the errors are linked to the first radio buttons